src/briefcase/commands/new.py

```diff
@@ -291,108 +377,130 @@ def build_context(
                 "briefcase_version": str(briefcase_version),
             }
         )
-        context.update(self.build_gui_context(context=context))
+        context.update(self.build_gui_context(context, project_overrides))
         return context
 
-    def build_app_context(self) -> dict[str, str]:
+    def build_app_context(self, project_overrides: dict[str, str]) -> dict[str, str]:
         """Ask the user for details about the app to be created.
 
         :returns: A context dictionary to be used in the cookiecutter project template.
         """
         formal_name = self.input_text(
-            intro="""
-First, we need a formal name for your application. This is the name that will
-be displayed to humans whenever the name of the application is displayed. It
-can have spaces and punctuation if you like, and any capitalization will be
-used as you type it.""",
+            intro=(
+                "First, we need a formal name for your application.\n"
+                "\n"
+                "This is the name that will be displayed to humans whenever the name "
+                "of the application is displayed. It can have spaces and punctuation "
+                "if you like, and any capitalization will be used as you type it."
+            ),
             variable="formal name",
             default="Hello World",
+            override_value=project_overrides.pop("formal_name", None),
         )
 
         # The class name can be completely derived from the formal name.
         class_name = make_class_name(formal_name)
 
         default_app_name = self.make_app_name(formal_name)
         app_name = self.input_text(
-            intro=f"""
-Next, we need a name that can serve as a machine-readable Python package name
-for your application. This name must be PEP508-compliant - that means the name
-may only contain letters, numbers, hyphens and underscores; it can't contain
-spaces or punctuation, and it can't start with a hyphen or underscore.
-
-Based on your formal name, we suggest an app name of '{default_app_name}',
-but you can use another name if you want.""",
+            intro=(
+                "Next, we need a name that can serve as a machine-readable Python "
+                "package name for your application.\n"
+                "\n"
+                "This name must be PEP508-compliant - that means the name may only "
+                "contain letters, numbers, hyphens and underscores; it can't contain "
+                "spaces or punctuation, and it can't start with a hyphen or "
+                "underscore.\n"
+                "\n"
+                "Based on your formal name, we suggest an app name of"
+                f"'{default_app_name}', but you can use another name if you want."
```

Comment:
There's a missing space here:
```suggestion
                "Based on your formal name, we suggest an app name of "
                f"{default_app_name!r}, but you can use another name if you want."
```